### PR TITLE
[DE-963] Build custom css for Unlayer editor

### PR DESCRIPTION
**Pull request by Cromeror**
Applied the **_inherit_** case for Unlayer tools name.

Related to: [DE-963](https://makingsense.atlassian.net/browse/DE-963)

[DE-963]: https://makingsense.atlassian.net/browse/DE-963?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +51,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-email-editor": "1.7.5",
-    "style-loader": "3.3.2",
+    "style-loader": "^3.3.2",
```
Comment (andresmoschini) [inline package.json:54]: Creo que no usamos `^` porque Renovate se encarga de actualizar las dependencias.

Comment (Cromeror) [reply package.json:54]: Dale, lo cambio

File: webpack.config.js [hunk 2]
```diff
@@ -51,10 +52,17 @@ module.exports = function (env) {
...
+                {
+                  test: /\.css$/i,
+                  use: [MiniCssExtractPlugin.loader, "css-loader"],
+                },
             ],
         },
         plugins: [
+            new MiniCssExtractPlugin({
+              filename: "static/[name].[contenthash].css"
+            }),
```
Comment (andresmoschini) [inline webpack.config.js:65]: 👍

File: src/customJs/main.css
Comment (andresmoschini) [inline src/customJs/main.css:1]: Nice!

**Pull request merged**